src/transformers/file_utils.py

```diff
@@ -263,6 +263,14 @@ def is_torch_cuda_available():
         return False
 
 
+def is_torch_fx_available():
+    if not is_torch_available():
+        return False
+    import torch
+
+    return version.parse(torch.__version__) >= version.parse("1.8")
+
```

Comment:
Let's move this logic out of this function since we only need to run it once and then we can save the results in a global var. e.g. untested below:

```
_torch_fx_available = False
try:
    import torch
    if version.parse(torch.__version__) >= version.parse("1.8"):
        _torch_fx_available = True

def is_torch_fx_available():
    return _torch_fx_available
```

Reply:
Good idea, it's done!